e2e/template.go

```diff
@@ -351,6 +351,7 @@ func baseTemplate() *datamodel.NodeBootstrappingConfiguration {
 		EnableNvidia:                   false,
 		EnableACRTeleportPlugin:        false,
 		TeleportdPluginURL:             "",
+		EnableArtifactStreaming:        true,
```

Comment:
is this something we want enabled by default for all e2e tests? I'd maybe lean more towards defaulting to false and having specific scenarios for this to be set to true within (though we do have a fair number of scenarios now... I've actually been thinking about adding the concept of "suites" which would be pre-configured scenario-sets designed to test certain things)

Reply:
compromise could also be that we start with false for now and later once this has rolled out we can default it to true

Reply:
Agreed. This is for ab e2e testing now -- PR is not yet ready for review.